pkg/workloads/cortex/client/cortex/binary/__init__.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from os import path
+import base64
+import sys
+import subprocess
+from typing import List, Optional, Tuple, Callable
+from io import BytesIO
+from cortex.exceptions import CortexBinaryException
+
+MIXED_CORTEX_MARKER = "~~cortex~~"
+
+
+def run():
+    """
+    Runs the CLI from terminal.
+    """
+    try:
+        process = subprocess.run([get_cli_path()] + sys.argv[1:], cwd=os.getcwd())
+    except KeyboardInterrupt:
+        sys.exit(130)  # Ctrl + C
+    sys.exit(process.returncode)
+
+
+def run_cli(
+    args: List[str],
+    hide_output: bool = False,
+    mixed: bool = False,
+) -> str:
+    """
+    Runs the Cortex binary with the specified arguments.
+
+    Args:
+        args: Arguments to use when invoking the Cortex binary.
+        hide_output: Flag to prevent streaming Cortex binary output to stdout. Defaults to False.
```

Comment:
Thinking we could get rid of the default param value comments (i.e. `"Defaults to False"`) as these will show up when hitting `shift+tab` anyway in notebooks and in python scripts.